### PR TITLE
Adding Hyper space to Repo

### PR DESCRIPTION
**Pull request by YanivHyper-Space**
### Description
This commit adds Hyperspace to opensearch-project. this is the first commits

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _partners/hyperspace.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/hyperspace_logo.png'
+
+# URL to the partner's website.
+link: 'https://www.hyper-space.io/'
+
+# logo_large is used on the extended partner details page.
+# It is normative to upload all extended partner page images to a subdirectory of /assets/media/partners/ with the name of the partner as the directory name.
+# For example given a partner named "SampleCompany" the logo_large image would be uploaded to /assets/media/partners/samplecompany/logo_large.png
+logo: '/assets/media/partners/hyperspace_logo_large.png'
```
Comment (nateynateynate) [inline _partners/hyperspace.md:18]: It looks like you might have forgotten to include hyperspace_logo_large.png with your PR.

**Review by nateynateynate — CHANGES_REQUESTED** [review]
I think you forgot to make `hyperspace_logo_large.png` part of this PR.

**Comment by pajuric** [comment]
@YanivHyper-Space - Thank you for submitting to become a partner. Please be sure to also submit this page to establish your own partner page on our site for inclusion in the partner finder: https://github.com/opensearch-project/project-website/issues/new?assignees=&labels=enhancement%2C+untriaged&projects=&template=partner_information.md&title=

**Comment by krisfreedain** [comment]
> @YanivHyper-Space - Thank you for submitting to become a partner. Please be sure to also submit this page to establish your own partner page on our site for inclusion in the partner finder: https://github.com/opensearch-project/project-website/issues/new?assignees=&labels=enhancement%2C+untriaged&projects=&template=partner_information.md&title=

@pajuric - since all that is already included in this PR, can we just open one on their behalf?

**Comment by nateynateynate** [comment]
@YanivHyper-Space - I think if you can amend this PR with the missing "large" logo we can get this merged in and get you listed!

File: _partners/hyperspace.md (hunk 1 above)
Comment (YanivHyper-Space) [reply _partners/hyperspace.md:18]: Hi!
Thanks for noticing. File added.

Thanks again

File: assets/media/partners/HS_Logo_-_400x400.png
Comment (krisfreedain) [inline assets/media/partners/HS_Logo_-_400x400.png:1]: @YanivHyper-Space - can you rename this to "hyperspace_logo_large.png" -- that is the filename you reference in the md file here "logo: '/assets/media/partners/hyperspace_logo_large.png'"

**Review by krisfreedain — COMMENTED** [review]
@YanivHyper-Space  file rename and this should be good to go!

**Review by nateynateynate — APPROVED** [review]

**Pull request merged**